src/rust/engine/pantsd/src/pantsd_testing.rs

```diff
@@ -32,15 +32,15 @@ pub fn launch_pantsd() -> (BuildRoot, OptionParser, TempDir) {
         .args(args)
         .env_clear()
         .envs(std::env::vars().filter(|(k, _v)| !k.starts_with("PANTS_")))
+        // NB: Set this to some unique sentinel value so its always consistent in tests
+        .env("_PANTS_VERSION_OVERRIDE", "3.0.0")
```

Comment:
One day there may be a Pants 3.0.0, so why not 99.99.99 or something very obviously fake?

Reply:
🤷‍♂️ I think no matter what it's an arbitrary value. Maybe 0.0.0 🙃

Reply:
Fine, I just like to use obviously fake values in tests, so you know what you're looking at while debugging.

Reply:
Oh hehe, we got a deprecation error anyways using `3.0.0`